Makefile

```diff
@@ -6,9 +6,15 @@ all: install
 fmt:
 	go fmt ./...
 
+# REBUILD touches all of the build-dependent source files, forcing them to be
+# rebuilt. This is necessary because the go tool is not smart enough to trigger
+# a rebuild when build tags have been changed.
+REBUILD:
+	@touch consensus/build*.go
```

Comment:
true, but I don't think there's really a better name for it, except `TouchBuildFilesToAlertGoToolThatObjectsNeedToBeRebuild`